### PR TITLE
Updating Media document

### PR DESCRIPTION
**Pull request by jpwalden**

### CODE REVIEW

File: Support/Help_Articles/Media/README.md [hunk 1]
```diff
@@ -87,6 +87,21 @@ NumberOfEntries=3
...
 
+# Which types of picture, audio & video files (media files) does Sailfish OS support
+
+The playback of media files discussed in this article requires the cooperation and resources of both the device and the Sailfish OS. Even if Sailfish OS supported certain media formats, it might be that the device hardware does not support it, which would result in a failure to use that media file.
+Not all media codecs are free but require licenses. This may prevent using them.
+ 
+## Multimedia formats compatible with Sailfish OS
+This is a list of codecs supported by Sailfish OS.
+* Audio and video formats currently supported by Sailfish are listed in this [document](/Reference/Core_Areas_and_APIs/Multimedia/).
+* Pictures: JPEG, PNG, GIF
```
Comment (jovirkku) [inline Support/Help_Articles/Media/README.md:98]: Also TIFF, BMP and even SVG. Tracker does not seem to catch SVG files, so they do not appear in Gallery, but they can be opened with the File Browser and viewed via the Gallery app.

File: Support/Help_Articles/Media/README.md [hunk 2]
```diff
@@ -87,6 +87,21 @@ NumberOfEntries=3
...
+
+Only the files compatible with these codecs appear in the Media (audio) and Gallery (video & photo) apps of Sailfish OS.
+ 
+## Troubleshooting
+If your Sailfish OS device does not seem to be able to use your media files in the expected way, please check first in the manual of your device that your device is capable of handling the type of media file you have at hand. The next thing to check is if the media database of Sailfish OS is somehow corrupted. Please seek help from this [document](/Support/Help_Articles/Tips_and_Tricks/#clearing-the-media-database).
+
```
Comment (jovirkku) [inline Support/Help_Articles/Media/README.md:104]: I wonder if line 110 (the 2nd related article) is needed now that the same linked information appears on line 97.

Comment (jpwalden) [reply Support/Help_Articles/Media/README.md:104]: Valid point, removed the duplicate information from the line 110.

File: Support/Help_Articles/Media/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/Media/README.md]: Thanks! Added that information

**Review by vigejolla — APPROVED** [review]

**Pull request merged**